elements/pl-checkbox/pl-checkbox.py

```diff
@@ -73,6 +74,16 @@ def prepare(element_html, data):
     if not (0 <= min_incorrect <= max_incorrect <= len_incorrect):
         raise ValueError('INTERNAL ERROR: incorrect number: (%d, %d, %d, %d)' % (min_incorrect, max_incorrect, len_incorrect, len_correct))
 
+    min_select = pl.get_integer_attrib(element, 'min-select', 1)
+    max_select = pl.get_integer_attrib(element, 'max-select', number_answers)
+
+    if min_select < 1:
+        raise ValueError('The attribute min-select is {:d} but must be at least 1'.format(min_select))
+    if min_select > max_select:
+        raise ValueError('min-select (%d) is greater than max-select (%d)' % (min_select, max_select))
+    if min_select > number_answers:
+        raise ValueError('min-select (%d) is greater than the total number of answers (%d)' % (min_select, number_answers))
```

Comment:
Should we add any checks for the relationship between `min/max-correct` and `min/max-select`? I think it doesn't make sense for `min-select > max-correct` or `max-select < min-correct`? I might be mistaken here, so please think about this! 😄

Reply:
Fixed. The code throws an error if `min-select > min-correct` or `max-select < max-correct` (I changed the conditions slightly).